apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -315,6 +315,17 @@
                 }
             }
         },
+        "channels" : {
+            "type" : "object",
+            "additionalProperties" : false,
+            "properties" : {
+                "max_count" : {
+                    "description" : "Max number of fsms allowed on node",
```

Comment:
We should not leak implementation-level details into the documentation. FSM is such a detail. Instead I'd call it `active state channels`.

Reply:
Yes, except there's a difference between serving only one-half of a state channel and having both halves running on the same node.

Reply:
That is correct, my suggestion was incomplete. Still I'd prefer to have a more high level approach.

Reply:
Suggestions are welcome, but we actually use "FSM" already in the published documentation, e.g.
[here](https://github.com/aeternity/protocol/blob/master/channels/OFF-CHAIN.md#channel_reestablish) and [here](https://github.com/aeternity/protocol/blob/master/node/api/channels_api_usage.md#authenticated-channel_create_tx). I agree that it's not a particularly user-friendly term.

Reply:
`active state channel clients` ?